packs/data/feats.db/agonizing-rebuke.json

```diff
@@ -11,7 +11,7 @@
             "value": null
         },
         "description": {
-            "value": "<p>When you terrorize your enemies, you also cause them painful mental distress. When you successfully @Compendium[pf2e.actionspf2e.Demoralize]{Demoralize} a foe, that foe takes 1d4 mental damage at the start of each of its turns as long as it remains @Compendium[pf2e.conditionitems.Frightened]{Frightened} and continues to engage in combat with you. If you have master proficiency in Intimidation, the damage increases to 2d4, and if you have legendary proficiency, the damage increases to 3d4.</p>"
+            "value": "<p>When you terrorize your enemies, you also cause them painful mental distress. When you successfully @Compendium[pf2e.actionspf2e.Demoralize]{Demoralize} a foe, that foe takes [[/r {1d4}[mental]]]{1d4 mental damage} at the start of each of its turns as long as it remains @Compendium[pf2e.conditionitems.Frightened]{Frightened} and continues to engage in combat with you. If you have master proficiency in Intimidation, the damage increases to [[/r {2d4}[mental]]]{2d4}, and if you have legendary proficiency, the damage increases to [[/r {3d4}[mental]]]{3d4}.</p>"
```

Comment:
Add a note to Intimidation on success for `action:demoralize`

Reply:
Done and updated re-en.json to match